virttest/utils_conn.py

```diff
@@ -1941,41 +1989,68 @@ def conn_setup(self):
             pattern_to_repl = {r".*sasl_allowed_username_list\s*=.*":
                                'sasl_allowed_username_list=%s' % sasl_allowed_username_list}
             self.client_libvirtdconf.sub_else_add(pattern_to_repl)
-        else:
+        elif not self.sasl_type == 'plain':
```

Comment:
This is not recommended coding style in python.  
`elif self.sasl_type != 'plain':` is better

Reply:
Will fix in next commit.

Reply:
Updated as suggested.